shared/applicability/cpes.yml

```diff
@@ -0,0 +1,324 @@
+centos6:
```

Comment:
Should still have this be `cpe:` and then a yaml lists of dictionaries.

Reply:
Why?

Reply:
Readability and understanding

Reply:
Right, I'm fine with that.